docs/README.md

```diff
@@ -59,6 +59,8 @@ If your server and services experience issues, feel free to come to [our support
 
 - [Self-building](self-building.md)
 
+- [Setting up services on the Matrix server configured with the MDAD playbook](setting-up-services-on-mdad-server.md)
```

Comment:
```suggestion
- [Setting up MASH services on a Matrix server configured with the matrix-docker-ansible-deploy Ansible playbook](setting-up-services-on-mdad-server.md)
```

I suppose this makes it clearer and avoids using the shortened version of the playbook's name (`MDAD`), which most users are probably not familiar with.

I'd also rename the `setting-up-services-on-mdad-server.md` file. Maybe something like: `setting-up-mash-services-on-matrix-docker-ansible-deploy-server.md`. It's quite long, but at least it's clear. You may have better ideas though.